web/src/pages/lesson1/types/example.ts

```diff
@@ -0,0 +1 @@
+export function test() {}
```

Comment:
Please add functionality to this function.

Reply:
Will do!